### PR TITLE
CAT-1001 Redesign and refactor physics standard project

### PR DESCRIPTION
**Pull request by Gorphi**

### CODE REVIEW

File: catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java [hunk 1]
```diff
@@ -161,167 +134,136 @@ public Project createStandardProject(String projectName, Context context) throws
 		Sprite rightVerticalWall = new Sprite("Right vertical wall");
 		Sprite rightBottomWall = new Sprite("Right bottom wall");
 
+		Sprite restart = new Sprite("Restart View");
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java]: "Restart Game" or something similar would be a better name for the sprite.

File: catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java [hunk 2]
```diff
@@ -347,10 +290,14 @@ public Project createStandardProject(String projectName, Context context) throws
 		return defaultPhysicsProject;
 	}
 
-	private Script createElement(Context context, String projectName, Sprite sprite, String fileName, int fileId, Vector2 position, float angle)
+	private Script createElement(Context context, String projectName, Sprite sprite, String fileName, int fileId, Vector2 position, float angle) throws IOException
+	{
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java]: To the previous line with you! :wink:

File: catroid/src/org/catrobat/catroid/utils/ImageEditing.java [hunk 3]
```diff
@@ -180,6 +182,27 @@ private static double calculateScaleFactor(int originalWidth, int originalHeight
 		}
 	}
 
+	public static Vector2 calculateXYScaleFactorToScreenSize(int resourceId, Context context) {
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/utils/ImageEditing.java]: Rename calculateXYScaleFactorToScreenSize or calculateScaleFactor to be consistent.

**Comment by Gorphi** [comment]
update String "Restart View" to "Restart Game" 
unify methode names in ImageEditing

File: catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java [hunk 4]
```diff
@@ -161,167 +134,137 @@ public Project createStandardProject(String projectName, Context context) throws
...
 
 		// Background
-		createElement(context, projectName, background, "background_480_800", R.drawable.background_480_800, new Vector2(), Float.NaN);
+		createElement(context, projectName, backgroundSprite, "physics_background_480_800",
+				R.drawable.physics_background_480_800, new Vector2(), Float.NaN);
 
 		// Ball
 		StartScript startScript = new StartScript();
-		startScript.addBrick(new SetGravityBrick(new Vector2(0.0f, -1.0f)));
+		startScript.addBrick(new SetGravityBrick(new Vector2(0.0f, -35.0f * (float)backgroundImageScaleVector.y)));
```
Comment (meiserm) [inline catroid/src/org/catrobat/catroid/common/standardprojectcreators/StandardProjectCreatorPhysics.java]: remove (float)backgroundImageScaleVector.y)

**Comment by meiserm** [comment]
Reviewed

**Pull request merged**